src/components/application_manager/CMakeLists.txt

```diff
@@ -447,6 +454,10 @@ target_link_libraries("AMHMICommandsLibrary" ${LIBRARIES} AMEventEngine)
 add_library("MessageHelper" ${MESSAGE_HELPER_SOURCES})
 target_link_libraries("MessageHelper" ${LIBRARIES})
 
+add_library("SystemTimeLibrary" ${SYSTEM_TIME})
```

Comment:
@AByzhynar not clear that in `${SYSTEM_TIME}` variable are *sources* related to system time. Bad variable name